src/Main.java

```diff
@@ -1,5 +1,71 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        double radius = 5.60;
+        double pi = Math.PI;
+        System.out.println(radius*radius*pi);
+
+        int A = 15;
+        int B = 3;
+        if ( (A > 10) && (A % 3 == 0) && (A < 10000) && (A % B == 0)) {
+            System.out.println(A + " is divisible by " + B);
+        }
+
+        if (A>B) {
+            System.out.println(A+B);
+        } else if (A<B) {
+            System.out.println(B-A);
+        } else {
+            System.out.println(A*B);
+        }
+
+        int x = 5;
+        int result = ((x + x + 6) / 2 ) - x;
+        if(result % 2 == 0) {
+        System.out.println("The result of ((x + x + 6) / 2 ) - x is divisible by 2");
+        } else {
+            System.out.println("The result of ((x + x + 6) / 2 ) - x is not divisible by 2");
+        }
+
+        A = 500;
+        B = 51;
+        int C = 11;
+        int D = 13;
+        int sum = A + B + C + D;
```

Comment:
you could calcutale sum in if, since the prerequisite for the total is the successfull pass of if